modules/ocf_apphost/manifests/init.pp

```diff
@@ -9,8 +9,17 @@
     cron => true;
   }
 
-  # useful for supervising user processes
-  package { 'daemontools':; }
+  # enable a persistent per-user systemd for all ocfdev users
+  file { '/var/lib/systemd/linger':
+    ensure  => directory,
+    # This makes sure only ocfdev users get per-user systemd. TODO: is this
```

Comment:
i think this sounds good.

only concern is that everyone still gets a user systemd instance when they log in (e.g. look at tsunami, there are tons of them; they just don't linger) so we need to be careful about spawning duplicate services. sounds like you have a solution for that already though.